CHT/SinkTemperature.C

```diff
@@ -18,7 +18,7 @@ T_(
     dataType_ = scalar;
 }
 
-void preciceAdapter::CHT::SinkTemperature::write(double * buffer)
+void preciceAdapter::CHT::SinkTemperature::write(double * buffer, bool provideMeshConnectivity)
 {
```

Comment:
Why no implementation here? Don't we need at least a warning?

Reply:
Done